### PR TITLE
[decoder] Add log option for bounding_box decoder @open sesame 10/10 17:02

### PR DESCRIPTION
**Pull request by anyj0527**
- Add an option to decide whether to log (print) result boxes.

An example:
```sh
...
** INFO: 16:50:53.507: Detect 8 boxes in 320 x 320 input image
** INFO: 16:50:53.507: [person] x:66 y:76 w:237 h:171 prob:0.8498
** INFO: 16:50:53.507: [tv] x:0 y:70 w:40 h:82 prob:0.7554
** INFO: 16:50:53.507: [chair] x:132 y:71 w:51 h:75 prob:0.6667
** INFO: 16:50:53.507: [laptop] x:63 y:181 w:124 h:66 prob:0.4536
** INFO: 16:50:53.507: [keyboard] x:63 y:218 w:65 h:32 prob:0.4395
** INFO: 16:50:53.507: [mouse] x:62 y:199 w:23 h:19 prob:0.4324
** INFO: 16:50:53.507: [book] x:0 y:181 w:24 h:26 prob:0.2654
** INFO: 16:50:53.507: [mouse] x:174 y:228 w:14 h:18 prob:0.2560
...
```

**Self evaluation:**
1. Build test: [X]Passed [ ]Failed [ ]Skipped
2. Run test: [X]Passed [ ]Failed [ ]Skipped

### CODE REVIEW

File: ext/nnstreamer/tensor_decoder/tensordec-boundingbox.c [hunk 1]
```diff
@@ -2104,6 +2135,10 @@ bb_decode (void **pdata, const GstTensorsConfig * config,
     goto error_unmap;
   }
 
+  if (bdata->do_log != 0) {
```
Comment (songgot) [inline ext/nnstreamer/tensor_decoder/tensordec-boundingbox.c:2138]: how about ```if (bdata->do_log)``` or ```if (bdata->do_log == 1)``` ?

**Review by songgot — APPROVED** [review]
LGTM

File: ext/nnstreamer/tensor_decoder/tensordec-boundingbox.c (hunk 1 above)
Comment (wooksong) [reply ext/nnstreamer/tensor_decoder/tensordec-boundingbox.c:2138]: @songgot ```do_log``` is not a boolean type variable, meaning only 0 can represent FALSE.

Comment (songgot) [reply ext/nnstreamer/tensor_decoder/tensordec-boundingbox.c:2138]: Okay, thank you.

**Review by myungjoo — APPROVED** [review]

**Pull request merged**